src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,78 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  max-width: 166px;
```

Comment:
The `max-width` property is not needed here. It's better to control the width of this element by its parent or by the content inside it. It's not a good practice to set width or max-width for blocks. Blocks should be flexible and occupy all the available horizontal space.